script.js

```diff
@@ -3,8 +3,301 @@ function displayMenu() {
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project3.png',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data5: {
+      title: 'Profesional Art<br>Printing Data More',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project4.jpg',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data6: {
+      title: 'Data Dashboard<br>Healthcare',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project5.png',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data7: {
+      title: 'Website Protfolio',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project6.jpeg',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+];
+
+let restProjects = '';
+let cardNumber = 0;
+let number = 0;
+for (let i = 2; i <= 6; i += 1) {
+  number = i + 1;
+  cardNumber = `data${number}`;
+  const titleCards = allData[i][cardNumber].title;
+  const mobileTitl = allData[i][cardNumber].mobileTitle;
+  const descriptionCards = allData[i][cardNumber].description;
+  const firstTool = allData[i][cardNumber].tools[0];
+  const secondTool = allData[i][cardNumber].tools[1];
+  const thirdTool = allData[i][cardNumber].tools[2];
+
+  restProjects += `<div class="pj11" id="n${i}">
+  <div class="cont_rest">
+    <h3 class="tproj1 mobile">${mobileTitl}</h3>
+    <button class="projects desktitbut" type="button" onclick="project${number}()">
+    <h3 class="tproj1 desktop" id="h3_2">${titleCards}</h3>
+  </button>
+    <p class="desproj1">${descriptionCards}</p>
+    <ul class="media2">
+      <li>
+        <div class="cont1">${firstTool}</div>
+      </li>
+      <li>
+        <div class="cont1">${secondTool}</div>
+      </li>
+      <li>
+        <div class="cont1">${thirdTool}</div>
+      </li>
+    </ul>
+    <div class="no_button">
+      <button class="bigbutton1 button21 projects" onclick='project${number}()' type="button">See Project</button>
+    </div>
+  </div>
+</div>`;
+}
+
+const container = document.getElementById('works');
```

Comment:
[ ]  kindly update the pop-up window's design to match the Figma template.
Figma template : 
![popdesign](https://user-images.githubusercontent.com/62156607/202281899-d01b138b-40a5-491a-b49e-c79c291dcbb4.PNG)

Your design : 
![designta3ou](https://user-images.githubusercontent.com/62156607/202281874-68eb05df-d779-4d27-9017-dfa4eeaf8028.PNG)

Reply:
Could you give me more information, please? Because in my pc it is working
[Grabación de pantalla desde 16-11-22 14:06:09.webm](https://user-images.githubusercontent.com/113067895/202283507-37572153-0d8d-4e3f-9769-3209d6151873.webm)

Reply:
@nedjwab Could you give me more information, please? Because in my pc it is working
[Grabación de pantalla desde 16-11-22 14:06:09.webm](https://user-images.githubusercontent.com/113067895/202293333-77b9cbde-faf1-41fb-9619-670b98d9f82f.webm)

Reply:
@nedjwab I have tried it on different a pc, and it's working. Could you give me more information? the browser that you are using, the size of the window, etc, please